docs/content/how-to/disconnected/known-issues.md

```diff
@@ -190,7 +190,7 @@ for isName in ${IMAGESTREAMS}; do
 done
 ```
 
-## IDMS/ICSP with only root registry are not propagated to the registry-override flag under ignition server
+## IDMS/ICSP with only root registry are filling to the registry-override flag under ignition server
```

Comment:
It made more sense with `not propagated` to me.  Not sure what is meant by `filling`.

Reply:
With "filling" I was meant that the --registries-override field in the different deployments, is filled with the registries-override content only when an exact match happens with an IDMS/ICSP exists with that record in the Management Cluster.